torch_sim/optimizers.py

```diff
@@ -1524,7 +1524,6 @@ def _ase_fire_step(  # noqa: C901, PLR0915
         dr_cell = cell_dt * state.cell_velocities
 
     # 6. Clamp to max_step
-    # Atoms
```

Comment:
In the ASE implementation rather than norm for these comparisons it uses `np.vdot` https://wiki.fysik.dtu.dk/ase/_modules/ase/optimize/fire.html. `np.vdot` always appears to return a scalar which means that our use of norm and theirs of vdot might be a problem?